troposphere/static/js/components/help/HelpPage.jsx

```diff
@@ -2,7 +2,62 @@ import React from "react";
 import stores from "stores";
 import globals from "globals";
 
+export default React.createClass({
+    displayName: "MarketplaceCorePage",
+    
+    render: function() {
+    
+    let imgs = ["https://info.massopencloud.org/wp-content/uploads/2016/03/bu-e1460748238801.jpg", "https://info.massopencloud.org/wp-content/uploads/2016/03/harvard.jpg", "https://info.massopencloud.org/wp-content/uploads/2016/03/mit.jpg", "https://info.massopencloud.org/wp-content/uploads/2016/03/northeastern.jpg"];
```

Comment:
good catch! I believe currently, we don' really need this page. I was using it as a placeholder for marketplace page.